R/html-tweak.R

```diff
@@ -251,38 +251,46 @@ tweak_homepage_html <- function(html, strip_header = FALSE) {
 
 # Mutates `html`, removing the badge container
 badges_extract <- function(html) {
-  # First try specially named div;
-  x <- xml2::xml_find_first(html, "//div[@id='badges']")
+  # First try specially named element;
+  x <- xml2::xml_find_first(html, "//*[@id='badges']")
+  force <- TRUE
 
   # then try usethis-readme-like paragraph;
   if (length(x) == 0) {
-    x <- xml2::xml_find_all(html, ".//*/comment()[contains(., 'badges: start')]/following-sibling::p[1]")
+    # Find all elements between the two comments:
```

Comment:
I think you can delete this line since it's saying the same thing as the next two lines